public/HTML/index.html

```diff
@@ -0,0 +1,251 @@
+                <div class="imgTextWrapper">
+                  <div class="squareImageWrapper grey-border"></div>
+                  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Perspiciatis totam distinctio obcaecati
+                    incidunt fugit ipsam nam odit minima suscipit deserunt?</p>
+                    <br>
+                  <button class="redButton"><a href="learn-more.html">LEARN MORE</button>
+                  <br>
+                  <hr>
+                  <br>
+                </div>
+              </div>
+            </div>
+          </div>
+
+        </div>
+      </div>
+
+    </div>
+
+  </section>
+
+<!--Footer Section-->
+  <section class="footer">
+
+    <div id="footerTitle">
+      <h2 class="sectionTitle">QUESTIONS OF COMMENTS? <strong>GET IN TOUCH!</strong></h2>
+    </div>
+
+<!--Footer Inputs-->
+    <div id="footerContent">
+      <div id="footer-form">
+
+        <div id="nameEmail">
+
+          <input class="nameEmailInput" type="text" placeholder="Name">
+
+          <input class="nameEmailInput" type="text" placeholder="Email"> <br>
+        </div>
+
+        <textarea class="footerFormArea" placeholder="Message" rows="4" cols="43"></textarea>
+        <br>
+        <br>
+        <button class="redButton">SEND MESSAGE</button>
+      </div>
+
+  <!--Right side contact info and logos-->
+      <div id="footer-content">
+        <p id="target">Lorem ipsum dolor sit amet consectetur adipisicing elit. Recusandae amet voluptatum eius
+          molestias odio magni consequuntur repellendus temporibus vitae doloremque.</p><br>
+        <div id="contactLinks">
+          <div id=rightContacts>
+            <div id="rightLogos">
+              <img class="contactLogos" src=../img/footerlink-home.png> <br>
+              <img class="contactLogos" src=../img/phone.png><br>
+              <img class="contactLogos" src=../img/footerlink-email.png><br>
+            </div>
+            <div id="rightInfo">
+              <p> 1234 Home<br>
+                Burlington, VT 05401<br>
+                USA</p>
+              <p>802-xxx-xxxx</p> <br>
+              <p>emaily@gmail.com</p> <br>
+            </div>
+          </div>
+          <div id="leftContacts">
+            <!--Build this section out-->
+            <div id="rightLogos">
+              <img class="contactLogos" src=../img/twitter.png> <br>
+              <img class="contactLogos" src=../img/instagram.png><br>
+              <img class="contactLogos" src=../img/dribbble.png><br>
+              <img class="contactLogos" src=../img/facebook.png><br>
+            </div>
+            <div id="rightInfo">
+              <p>@twitteringabout</p><br>
+              <p>instagram.com</p> <br>
+              <p>dribble.com</p> <br>
+              <p>facebook.com</p> <br>
+            </div>
+          </div>
+          <div class="contact">
+
+          </div>
+
+        </div>
+      </div>
+
+    </div>
+    </div>
+

```

Comment:
Super nice footer!  I know you were excited about it, and for good reason! Three small things to note : 
1) Your icons lack the full spacing within the footer itself
2) The SEND MESSAGE button doesn't line up with the icons in the bottom of the div.
     - You can draw a straight line from the bottom of the button to the bottom of the icons in the example.
3) Your NAME/EMAIL input fields aren't properly sized.  They should span the width of the text area and have some space in between.

Small things.  It looks great.  These are little things that, should you revisit, will give you a nice little bump!